healthcheck.js

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+'use strict'
+require('dotenv').config()
+
+const fetch = require('node-fetch')
+
+const signale = require('./src/utils/signale')
+const checkMongoDB = require('./src/utils/connect')
+
+const port = process.env.ACKEE_PORT || process.env.PORT || 3000
+const dbUrl = process.env.ACKEE_MONGODB || process.env.MONGODB_URI
+const serverUrl = `http://localhost:${ port }`
+
+if (dbUrl == null) {
+	signale.fatal('MongoDB connection URI missing in environment')
+	process.exit(1)
+}
+
+const checkHTTP = async (url) => {
+	return await fetch(url)
+		.then((res) => {
+			if (res.status < 200 || res.status >= 300) {
+				new Error(`Status Code: ${ res.status }`)
+			}
+
+			return res.text() // read the body so it can be garbage collected
```

Comment:
Curious about the "read the body so it can be garbage collected". Is that really the case?

Reply:
At least a couple of years ago, it was. https://github.com/node-fetch/node-fetch/issues/83
I keep it in just to be on the safe side. Usually could just use a `HEAD` request, but the server isn't listened for `HEAD`